docs/team/merlinlim.md

```diff
@@ -0,0 +1,51 @@
+---
+layout: page
+title: Merlin Lim's Project Portfolio Page
+---
+
+## Project: Productiv
+
+Productiv is a desktop application for product managers to organise their product-related information. The user interacts with it using a CLI, and it has a GUI created with JavaFX. It is written in Java, and has about 10 kLoC.
+
+Given below are my contributions to the project.
+
+* **New Feature**: Delete command for Meeting
+    * Delete command for meeting allows users to delete meeting from list.
+    * This feature allows users to remove meetings which may have been done or cancelled
+* **New Feature**: Find command for Meeting
+    * Find command for meeting allows users to find a meeting based on a keyword.
+    * This feature allows users to search for a specific meeting more conveniently. 
+* **New Feature**: Edit command for meeting
+    * Edit command for meeting allows users to edit meeting from list
+    * This feature allows users to edit a specific meeting when details have changed (e.g Time, Location, etc)
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=merlinlim)
+
+* **Project management**:
+  * Managed releases `v1.3` - `v1.5rc` (3 releases) on GitHub
+
+* **Enhancements to existing features**:
+  * 
+
+* **Documentation**:
+  * User Guide:
+    * Updated documentation for feature `meeting`
+    * Include a Glossary for meeting. 
```

Comment:
All first words should be past tense. Here and elsewhere ("included", "stardardized", "updated", etc.)